Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:14.04
```

Comment:
Well you will still need to the `python-scipy` package, so assuming you can install it, this should work.